completion/available/docker-compose.completion.bash

```diff
@@ -676,7 +676,7 @@ _docker_compose() {
 	done
 
 	local completions_func=_docker_compose_${command//-/_}
-	declare -F $completions_func >/dev/null && $completions_func
+	_command_exists $completions_func && $completions_func
```

Comment:
Since this was specifically testing for being a function, you might want to use [`_is_function`](https://github.com/Bash-it/bash-it/blob/master/lib/helpers.bash#L167)